Bottom: dbf2268a62da8784c6ab9033fc297c6d57babbda
Top:    d6059001f0d2f622720ad871c89e3c1ef4410125
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-27 23:56:47 -0700

Refresh of rs-1sge

---

diff --git a/src/rsocket.c b/src/rsocket.c
index e480bfd..5be242f 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1129,7 +1129,7 @@ ssize_t rsend(int socket, const void *buf, size_t len, int flags)
 		if (xfer_size > rs->target_sgl[rs->target_sge].length)
 			xfer_size = rs->target_sgl[rs->target_sge].length;
 		if (xfer_size > rs_sbuf_left(rs)) {
-			if (xfer_size <= rs->sbuf_bytes_avail - rs_sbuf(left(rs)))
+			if (xfer_size <= rs->sbuf_bytes_avail - rs_sbuf_left(rs))
 				rs->ssge.addr = (uintptr_t) rs->sbuf;
 			else
 				xfer_size = rs_sbuf_left(rs);
